muselsl/viewer_eeg_v1.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
```

Comment:
I think since we'll focus on making the same viewer work for all data types renaming these viewer files is unnecessary.